src/bd2k/util/strings.py

```diff
@@ -2,6 +2,31 @@
 
 import inspect
 
+def to_english(input_list, separator=",", conjuction="and"):
+    """
+    convert list to a more natural and readable string. If given empty list,
+    returns python's default empty string representation: [].
+
+    :param input_list: list of any type, as long as it can be cast to string
+    :param separator: char (or string) to insert between elements in list
+    :param conjuction: string used to connect the final element in list
+    :return: string
+
+    ex:
+    >>> to_english(['a','b','c'])
+    "a, b, and c "
+    """
+    if len(input_list)==0:
+        return str(input_list)
```

Comment:
As a user, I might want a way to override this behavior. Could we have an optional parameter called `empty` that defaults to '[]'.